src/server-actions/Reward/Reward.ts

```diff
@@ -0,0 +1,44 @@
+import { InsertReward, InsertUserReward } from "@/src/db/schema"
+import { CreateServerAction } from ".."
+import { AddReward, getRewardsById } from "@/src/db/data-access/reward/query"
+import {
+  AddRewardForUser,
+  GetRewardIdsForUser
+} from "@/src/db/data-access/user-reward/query"
+
+export const AddRewardAction = CreateServerAction(
+  false,
```

Comment:
should be true everything on profile should be authenticated 
expect some of the get ones